plans/feat-create-mulmoclaude-plugin-cli.md

```
@@ -0,0 +1,146 @@
+npx create-mulmoclaude-plugin notes-plugin
+npx create-mulmoclaude-plugin @example/cool-plugin
+```
+
+Validates against npm package-name rules (lowercase, optional `@scope/`
+prefix, hyphens, no spaces, length ≤ 214). Refuses to overwrite an
+existing directory. The package name in `package.json` is set to the
+argument verbatim; everything else stays as the counter sample.
+
+No interactive prompts (Phase 1). The CLI prints next steps:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)